src/core/datastore/repositories/network-repository.js

```diff
@@ -22,8 +23,12 @@ import { buildCollectionUrl } from './utils';
 
 export class NetworkRepository extends Repository {
   read(collection, query, options = {}) {
+    if (options.useDeltaSet) {
+      return deltaSet(collection, query, options);
```

Comment:
I believe this returns only the modified entities. I don't think this is the spec for delta set in CacheStore.find(). Again, I don't have an app to test it, just judging from the code.

Reply:
You were right. It now returns what matches the provided query in cache after the requests are complete.